pkg/collector/corechecks/ebpf/probe/oomkill/oom_kill.go

```diff
@@ -14,18 +14,17 @@ package oomkill
 import (
 	"fmt"
 	"math"
-	"unsafe"
 
 	"golang.org/x/sys/unix"
 
 	manager "github.com/DataDog/ebpf-manager"
-	bpflib "github.com/cilium/ebpf"
 
 	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/ebpf/probe/ebpfcheck"
 	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/ebpf/probe/oomkill/model"
 	"github.com/DataDog/datadog-agent/pkg/ebpf"
 	"github.com/DataDog/datadog-agent/pkg/ebpf/bytecode"
 	"github.com/DataDog/datadog-agent/pkg/ebpf/bytecode/runtime"
+	ebpfutil "github.com/DataDog/datadog-agent/pkg/ebpf/util"
```

Comment:
I didn't realize we put it into a `util` package. In general, I prefix to avoid packages named like that because they just become dumping grounds with no clear purpose and end up importing a ton of packages.

Any reason it cannot be in the base `pkg/ebpf` package?

Reply:
Alternatively, you could create `pkg/ebpf/map` package

Reply:
Alternatively you could create `pkg/ebpf/maps` package, but that would conflict with `golang.org/x/exp/maps`

Reply:
I see now that you moved it in this PR. My comments still apply.

Reply:
Done, moved to maps package